websites/G/GenoAnime/presence.ts

```diff
@@ -0,0 +1,145 @@
+const presence = new Presence({
+  clientId: "810203651317432351"
+  }),
+  strings = presence.getStrings({
+      play: "presence.playback.playing",
+      pause: "presence.playback.paused"
+  }),
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+let timestamps: number[],
+  video: HTMLVideoElement,
+  currentTime: number,
+  duration: number,
+  paused: boolean,
+  iFrameVideo: boolean,
+  playback: boolean,
+  anime_breadcumb: string;
+
+presence.on(
+  "iFrameData",
+  (data: {
+      iframe_video: {
+          duration: number;
+          iFrameVideo: boolean;
+          currTime: number;
+          paused: boolean;
+      };
+  }) => {
+      playback = data.iframe_video.duration !== null ? true : false;
+      if (playback) {
+          iFrameVideo = data.iframe_video.iFrameVideo;
+          currentTime = data.iframe_video.currTime;
+          duration = data.iframe_video.duration;
+          paused = data.iframe_video.paused;
+      }
+  }
+);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+          largeImageKey: "genoanime",
+          startTimestamp: browsingStamp
+      },
+  title = document.title.slice(0, -13); //title of the page
+  if (document.location.pathname == "/") {
+      presenceData.details = "Exploring Genoanime";
+  } else if (document.location.pathname.includes("/browse")) {
+      presenceData.details = "Exploring Genoanime library";
+      presenceData.buttons = [{
+          label: "View Library",
+          url: document.location.href
+      }];
+  } else if (document.location.pathname.includes("/details")) {
+      presenceData.details = `Checking Synopsis`;
+      presenceData.state = document.querySelector(
+          ".anime__details__title h3"
+      ).textContent;
+      anime_breadcumb = String(
+          document.querySelector(
+              "#container > section > div > div.anime__details__content > div > div.col-lg-9 > div > div.anime__details__btn > a.watch-btn"
+          )
+      );
+      presenceData.buttons = [{
+              label: "Watch It",
+              url: String(anime_breadcumb)
+          },
+          {
+              label: "Check Synopsis",
+              url: document.location.href
+          }
+      ];
+  } else if (document.location.pathname.includes("/watch")) {
+      presenceData.details = title;
+      presenceData.state =
+          "Episode " + String(document.location.href.split("episode=")[1]);
+      anime_breadcumb = String(document.querySelector("#anime_details_breadcrumbs"));
```

Comment:
Um yeah what? You don't need to string it.
```suggestion
          "Episode " + document.location.href.split("episode=")[1];
      anime_breadcumb = document.querySelector<HTMLAnchorElement>("#anime_details_breadcrumbs").href;
```